diff --git a/printerInterface.py b/printerInterface.py
index 69211b1..334a6d4 100644
--- a/printerInterface.py
+++ b/printerInterface.py
@@ -250,10 +250,10 @@ class PrinterData:
 	CORP_WEBSITE_E = "https://www.klipper3d.org/"
 
 	def __init__(self, API_Key, URL='127.0.0.1'):
-		self.op = MoonrakerSocket(URL, 80, API_Key)
+		self.op = MoonrakerSocket(URL, <port>, API_Key)
 		self.status = None
 		print(self.op.base_address)
-		self.ks = KlippySocket('/tmp/klippy_uds', callback=self.klippy_callback)
+		self.ks = KlippySocket('/run/klipper/api', callback=self.klippy_callback)
 		subscribe = {
 			"id": 4001,
 			"method": "objects/subscribe",
@@ -363,9 +363,9 @@ class PrinterData:
 			return
 		self.update_variable()
 		#alternative approach
-		#full_version = self.getREST('/printer/info')['result']['software_version']
-		#self.SHORT_BUILD_VERSION = '-'.join(full_version.split('-',2)[:2])
-		self.SHORT_BUILD_VERSION = self.getREST('/machine/update/status?refresh=false')['result']['version_info']['klipper']['version']
+		full_version = self.getREST('/printer/info')['result']['software_version']
+		self.SHORT_BUILD_VERSION = '-'.join(full_version.split('-',2)[:2])
+		#self.SHORT_BUILD_VERSION = self.getREST('/machine/update/status?refresh=false')['result']['version_info']['klipper']['version']
 
 		data = self.getREST('/printer/objects/query?toolhead')['result']['status']
 		toolhead = data['toolhead']
diff --git a/run.py b/run.py
index d5f3612..be18526 100755
--- a/run.py
+++ b/run.py
@@ -4,7 +4,7 @@ from dwinlcd import DWIN_LCD
 encoder_Pins = (26, 19)
 button_Pin = 13
 LCD_COM_Port = '/dev/ttyAMA0'
-API_Key = 'eb56bb488d3143708656f60074f70af0'
+API_Key = '<apikey>'
 
 DWINLCD = DWIN_LCD(
         LCD_COM_Port,
